### DISCUSSION TITLE
Shelly Z-Wave

### DISCUSSION THREAD

**Discussion by Borogarth** [camueller/SmartApplianceEnabler#620] [Q&A]
Hi,
funktionieren die Shelly Z-Wave Produkte auch mit den SAE und SMA Visualisierung?
Ich finde den niedrigen Standby verbrauch doch recht spannend langfristig.
LG

**Comment by camueller** [comment]
Bei den Z-Wave Produkten steht ja immer dabei

'Ein Gateway (Smart Hub) ist nicht im Lieferumfang enthalten. Ein Gateway (Smart Hub) ist erforderlich für die Konfiguration des Geräts.'

Wenn dieser Hub die Verbrauchsdaten per HTTP liefern kann, sollte es funktionieren. Ob das so ist, musst Dir jemand anderes beantworten bzw. Du musst das in anderen Foren herausfinden.
